src/components/horizontal-card/HorizontalCard.jsx

```diff
@@ -19,10 +14,10 @@ const HorizontalCard = ({
           <span className="card-sub-heading text-gray">{description}</span>
           <div className="price">
             <span className="card-sub-heading text-md">
-              ₹{new Intl.NumberFormat("en-IN").format(price)}
+              ₹{new Intl.NumberFormat("en-IN").format(discountedPrice)}
```

Comment:
Wahh didn't know about this number formatting look cool 🔥

Reply:
Yes, Abhishek told me to use it in one of the PR review.